packages/IMAPClient-UI.package/ICAddressBookDialog.class/instance/updateNewEmailAddress..st

```diff
@@ -0,0 +1,6 @@
+toolbuilder
+updateNewEmailAddress: aString
```

Comment:
update reads like you are editing an existing adress, but this method seems to create a new entry.

Reply:
If you check the following calls you will see it works as intended.